test/server/test_chat_index.ts

```
@@ -0,0 +1,392 @@
+      }),
+    });
+    assert.ok(entry);
+    assert.equal(entry?.roleId, "general");
+  });
+});
+
+describe("indexStale", () => {
+  it("respects the limit", async () => {
+    // Plant several stale sessions and assert only `limit` get processed.
+    const ts = Date.now();
+    const ids: string[] = [];
```

ESLint

[error] Either use this collection's contents or remove the collection.

(sonarjs/no-unused-collection)